src/validation.cpp

```diff
@@ -326,7 +326,7 @@ void CChainState::MaybeUpdateMempoolForReorg(
     // previously-confirmed transactions back to the mempool.
     // UpdateTransactionsFromBlock finds descendants of any transactions in
     // the disconnectpool that were added back and cleans up the mempool state.
-    const uint64_t ancestor_count_limit = gArgs.GetIntArg("-limitancestorcount", DEFAULT_ANCESTOR_LIMIT);
+    const uint64_t ancestor_count_limit = gArgs.GetIntArg("-limitancestorcount", ::ancestorLimit);
```

Comment:
Maybe naive question: what is the purpose of having an indirection through `ancestorLimit` (or others) instead of directly using the constants?

Reply:
Right, I don't get it either why this is changed.